contracts/contracts/token/OUSD.sol

```diff
@@ -45,7 +45,11 @@ contract OUSD is Initializable, InitializableToken, Governable {
     uint256 public nonRebasingCredits;
     uint256 public nonRebasingSupply;
     mapping(address => uint256) public nonRebasingCreditsPerToken;
-    mapping(address => bool) public rebaseOptInList;
+    // Uint8 represents opt in state
+    // -1 explicit opt out
+    // 0 (default) not set (contracts by default opt on, EOAs by default opt in)
```

Comment:
nit: did you mean contract by default opt out (vs "on")?

Reply:
I've updated this.